.circleci/config.yml

```diff
@@ -344,7 +344,7 @@ workflows:
           name: helm_lint
           env: 'staging'
       # Docker builds
-      - hmpps/build_docker:
+      - build-docker-image:
           <<: *only_master
           name: build-docker-image-master
```

Comment:
Do we also need to add back in the `staging: 'true'` line that was removed by [this commit](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/commit/aa62571d3e07e6f5a22eb9c1518a61f376396cf6#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L347)?

Reply:
Yes, great catch! Added back!